.devcontainer/devcontainer.json

```diff
@@ -41,6 +41,6 @@
     ],
     // Mount the parent as /workspaces so we can pip install peers as editable
     "workspaceMount": "source=${localWorkspaceFolder}/..,target=/workspaces,type=bind",
-    // After the container is created, install the python project in editable form
-    "postCreateCommand": "pip install $([ -f dev-requirements.txt ] && echo '-c dev-requirements.txt') -e '.[dev]' && pre-commit install"
+    // After the container is created, install the uv env
+    "postCreateCommand": "uv pip install $([ -f dev-requirements.txt ] && echo '-c dev-requirements.txt') -e '.[dev]' && pre-commit install"
```

Comment:
Must: Not sure if this has already been discussed but needing to prefix every `pip` command with `uv` is a pretty large breaking change. Do we want to commit to it and make a major release or is there some aliasing we could do? @GDYendell and @coretl

Reply:
yeah `uv pip` works exactly the same but is waaaay faster. Worst case you just alias it?